src/Altinn.App.Core/Internal/Process/IProcessNavigator.cs

```diff
@@ -0,0 +1,20 @@
+using Altinn.App.Core.Internal.Process.Elements.Base;
+using Altinn.Platform.Storage.Interface.Models;
+
+namespace Altinn.App.Core.Internal.Process
+{
+    /// <summary>
+    /// Interface used to descipt the process navigator
+    /// </summary>
+    public interface IProcessNavigator
```

Comment:
Is IProcessEngine considered as a home for this method since this has process navigation methods already?